app/backend/app.py

```diff
@@ -102,7 +102,6 @@
 
 @bp.route("/")
 async def index():
-    # TODO: use msal loginRedirect on a blankish page
```

Comment:
Meant to remove this TODO in the built-in auth PR but forgot to push the removal commit before merging. This TODO isn't actually needed, since built-in auth works.